mnist-cnn/label_errors_mnist_train_cnn.ipynb

```diff
@@ -2266,10 +2267,37 @@
     "np.random.seed(4)\n",
     "\n",
     "# Run cross-validation\n",
-    "cj, psx = cleanlab.count.estimate_confident_joint_and_cv_pred_proba(X_train, y_train, clf=cnn, )\n",
+    "cj, pred_probs = cleanlab.count.estimate_confident_joint_and_cv_pred_proba(X_train, y_train, clf=cnn, )\n",
     "est_py, est_nm, est_inv = cleanlab.count.estimate_latent(cj, y_train)"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "These are the different cleanlab methods for selecting noise, for more details view [`cleanlab/filter.py`](https://docs.cleanlab.ai/v2.0.0/cleanlab/filter.html)."
```

Comment:
We want to start moving to stable/ links instead:  https://docs.cleanlab.ai/stable/cleanlab/filter.html

Generally be careful about linking cleanlab things from inside Examples/ because links can easily become outdated.